src/test/kotlin/uk/gov/justice/digital/hmpps/hmppshdcapi/licences/LicenceServiceTest.kt

```diff
@@ -152,15 +279,57 @@ class LicenceServiceTest {
         "08:00",
       ),
     )
-    assertThat(result?.curfewHours).isEqualTo(
-      CurfewHours(
-        "20:00", "08:00",
-        "20:00", "08:00",
-        "20:00", "08:00",
-        "20:00", "08:00",
-        "20:00", "08:00",
-        "20:00", "08:00",
-        "20:00", "08:00",
+    assertThat(result?.curfewTimes).isEqualTo(
```

Comment:
**question:** do we need to test this list for every scenario? 
is there anything about the individual thing we are trying to test for that would make the answer different?

If not then it might be overkill repeatedly test (and have to maintain these)

Reply:
The `licenceId` does change for each set curfew times depending on address scenario, but maybe it's not worth testing for this?

Reply:
I should prob remove subsequent the asserts for firstNightCurfewHours too as they don't change depending on the scenario.

Reply:
`licenceId` removed as the curfew times that are returned as part of the HDC licence will always belong to the surrounding HDC Licence it is a part of. Mistakenly added following separate CVL DB implementation detail where `licenceId` is part of the returned curfew times object from the CVL DB.